src/test/java/io/druid/guice/JsonConfiguratorTest.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright 2015 Metamarkets Group, Inc.
```

Comment:
Yep, my copyright config between Druid and Druid-api was not consistent. Should be fixed now.